src/core/router/helpers.js

```diff
@@ -26,7 +29,8 @@ const createRouteGuard = (to, from, next) => {
 };
 
 const accessWalletProps = route => {
-  const walletType = route.query && route.query.type ? route.query.type : '';
+  const walletType =
+    route.query && route.query.type ? xss(route.query.type) : '';
```

Comment:
can we create one func to pass all query params through xss, this is currently repetitive